admin/controllers/administrator.js

```diff
@@ -6,6 +6,11 @@ const settingsErrorMessages = require('../lib/errors/settings')
 const settingsValidator = require('../lib/validator/settings-validator')
 const checkWindowValidator = require('../lib/validator/check-window-validator')
 const checkWindowErrorMessages = require('../lib/errors/check-window')
+const {
+  fetchCheckWindow,
```

Comment:
I thought we had agreed not to destructure service calls?  It reads more clearly if we leave it in - the service clearly identifies the layer the code is working at.

Reply:
OK, will modify this.